src/net_processing.cpp

```diff
@@ -6295,6 +6295,7 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
         //
         // Message: getdata (transactions)
         //
+        {
```

Comment:
style nit: Indenting the body of the new scope would be nice

Reply:
done